modules/persistent-storage-csi-group-snapshots-create.adoc

```diff
@@ -0,0 +1,134 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-group-snapshots.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-group-snapshots-create_{context}"]
+= Creating a volume group snapshot
+
+When you create a `VolumeGroupSnapshot` object, {product-title} creates a volume group snapshot.
+
+.Prerequisites
+* Logged in to a running {product-title} cluster.
+* The persistent volume claims (PVCs) that you want to group for the snapshot have been created using a CSI driver that supports `VolumeGroupSnapshot` objects.
+* A storage class to provision the storage back end.
+* Administrator has created the `VolumeGroupSnapshotClass` object.
+* No pods are using the PVC that you want to take a snapshot of.
```

Comment:
we don't support group snapshots for in use volumes?
@mulbc @gnufied @jsafrane

Reply:
it's no different than regular snap so while the warning statement is correct if the app is quiesced it should be ok